lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -533,29 +536,35 @@ class MailboxController extends BaseMailboxController {
     }
   }
 
-  void pressMailboxSelectionAction(BuildContext context, MailboxActions actions,
-      List<PresentationMailbox> selectionMailbox) {
+  void pressMailboxSelectionAction(
+      BuildContext context, 
+      MailboxActions actions,
+      List<PresentationMailbox> selectedMailboxList
+  ) {
     switch(actions) {
       case MailboxActions.delete:
-        _openConfirmationDialogDeleteMailboxAction(context, selectionMailbox.first);
+        _openConfirmationDialogDeleteMultipleMailboxAction(context, selectedMailboxList);
         break;
       case MailboxActions.rename:
-        _openDialogRenameMailboxAction(context, selectionMailbox.first);
+        _openDialogRenameMailboxAction(context, selectedMailboxList.first);
```

Comment:
are you sure the button `rename` will be not enable when select multiple mailbox?

Reply:
sure. please read code.

Reply:
i only remind you